src/Sound.js

```diff
@@ -180,14 +180,16 @@ export default function Sound(props) {
           </div>
         </>
       )}
-      <SoundList
-        header="Pack"
-        tracks={packSounds}
-        selectedTrack={sound?.id || packSounds[0]?.id || 0}
-        setSelectedTrack={() => {}}
-        onSoundClick={() => window.scrollTo(0, 0)}
-        className="pb-16"
-      />
+      {packSounds[0] && (
```

Comment:
nit: we could do `packSounds.length && ...`. This is more idiomatic in JS since `0` is falsy.
```suggestion
      {packSounds.length && (
```

Reply:
Oh! So, if `packSounds[0]` has a falsy value, the "Pack" `<SoundList />` could still not render, even though it's not empty?

Reply:
It appears that if `packSounds` is empty, like in https://freesound-player.vercel.app/sound/396855, the application displays `packSounds.length`, which would be `0` in this case, onscreen where the Pack `<SoundList />` component would be if `packSounds` was nonempty.

It works fine for packs that are not empty, though, like https://freesound-player.vercel.app/sound/376626. The issue persists across both Chrome and Safari.

![Screen Shot of sound with empty pack at 2020-11-20 at 6 55 16 PM](https://user-images.githubusercontent.com/43425812/99865914-f0667280-2b61-11eb-8d17-afc7d9f27783.png)

![Screen Shot of sound with nonempty pack at 2020-11-20 at 6 56 34 PM](https://user-images.githubusercontent.com/43425812/99865941-1e4bb700-2b62-11eb-9ff3-c1d6a2bc43ea.png)

It works perfectly fine if we change [lines 183 though lines 192 of the latest commit at the time of this writing](https://github.com/amilajack/freesound-player/blob/e1bfac95ed48a042997dafa12233bd9ae3067755/src/Sound.js#L183) to

```
      {packSounds.length ? (
        <SoundList
          tracks={packSounds}
          selectedTrack={sound?.id || packSounds[0]?.id || 0}
          setSelectedTrack={() => {}}
          onSoundClick={() => window.scrollTo(0, 0)}
          header="Pack"
          className="pb-16"
        />
      ) : (
        <></>
      )}
```

but I'm wondering if the original approach would just be cleaner. What are your thoughts?